examples/inventory.ini.example

```diff
@@ -0,0 +1,37 @@
+[cvp:children]
+development
+lab
+
+[development]
+cvp_foster  ansible_httpapi_port=443
```

Comment:
Should we change these to be the official version names not the internal names?

Reply:
Sure we can. In the meantime, since we can consume collection as-is, we could reserve this folder to provide real examples and put all the testing into `ansible_collections/arista/cvp/tests/` directory. Does it make sense ?